### DISCUSSION TITLE
Surface Pro 4 touch screen works but pen doesnt

### DISCUSSION THREAD

**Discussion by dwiggins4121** [linux-surface/linux-surface#1409] [Q&A]
So i followed the instructions for Debian and got the touch screen working, multi-touch also works.
uname -r has surface listed but the pen does not work.

if any has had the same issue but got it resolved, help would be appreciated. Thanks

**Comment by djones171** [comment]
Did you try replacing the battery in the pen?  It is a AAAA battery, and you just twist the darker grey top to get access to the battery.  Hope this helps. FYI, my pen works on Surface Pro 4. -Don
